### PR TITLE
[fr] QUE_AVOIR-improved

### PR DESCRIPTION
**Pull request by GillouLT**
Related to loops: https://github.com/languagetooler-gmbh/task-force-french/issues/90

QUE_AVOIR[1]: AP added with example
QUE_AVOIR[6]: AP added with example (similar to QUE_AVOIR[1])
QUE_AVOIR[10]: Slightly improved APs and new AP added based on loop-finder entries (mainly ugly FPs), rule name is name="det + n f s suivi par . p"
Other QUE_AVOIR rules: Fixed empty quotation marks in suggestions, apparently the patterns have changed over time but the token positions in the suggestions haven't been adapted to the new pattern

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -96630,16 +96639,33 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
...
+                    <token postag="V.* ppa.* s?p|V.* ppa.* m sp?" postag_regexp="yes"/>
+                    <example>Je ne récolte que la sève des arbres que j'ai moi-même plantés.</example>
                 </antipattern>
                 <pattern>
                     <token negate="yes" regexp="yes">plus|moins|avec|et|dans|de|d'|au|à</token>
                     <token postag="D f s"/>
                     <token postag="J f s|C coor|A" postag_regexp="yes" min="0" max="4"/>
                     <token postag="N f s">
-                        <exception regexp="yes">semaine|manière|année|raison</exception>
+                        <exception regexp="yes">collection|semaine|manière|année|raison|série|ensemble|gamme|assortiment|lot|palette|variété|choix|répertoire|éventail</exception>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:96668]: Still QUE_AVOIR[5], I found a FP with "collection" so I went for more words related to sets

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -96544,7 +96544,11 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token postag="V.* ppa.* s?p|V.* ppa.* m sp?" postag_regexp="yes"/>
                     <token regexp="yes">de|d'|à</token>
                     <token postag="R pers obj.*" postag_regexp="yes" min="0" max="3"/>
+                    <token postag="R pers suj 3 e s" min="0" max="1"/>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:96547]: QUE_AVOIR[10] changes start here

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 3]
```diff
@@ -95850,6 +95850,18 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
...
+                    <token regexp="yes">ces|tes|mes|ses</token>
+                    <token postag="J.*|A" postag_regexp="yes" min="0" max="3"/>
+                    <token postag="C coor" postag_regexp="yes" min="0" max="1"/>
+                    <token postag="J.*|A" postag_regexp="yes" min="0" max="3"/>
+                    <token postag="N [em] p" postag_regexp="yes" skip="3"/>
+                    <token regexp="yes" skip="2">que|qu'</token>
+                    <token postag="V avoir (ind|con|sub).*" postag_regexp="yes" skip="2"/>
+                    <token postag="V.* ppa.* sp?|V.* ppa.* f s?p" postag_regexp="yes"/>
+                    <example>Ce n'est pas pour tes beaux yeux que j'ai fait cela.</example>
+                </antipattern>
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:95864]: QUE_AVOIR[1]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 4]
```diff
@@ -96630,16 +96664,33 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
...
+                    <token postag="V.* ppa.* s?p|V.* ppa.* m sp?" postag_regexp="yes"/>
+                    <example>Je ne récolte que la sève des arbres que j'ai moi-même plantés.</example>
                 </antipattern>
                 <pattern>
                     <token negate="yes" regexp="yes">plus|moins|avec|et|dans|de|d'|au|à</token>
                     <token postag="D f s"/>
                     <token postag="J f s|C coor|A" postag_regexp="yes" min="0" max="4"/>
                     <token postag="N f s">
-                        <exception regexp="yes">semaine|manière|année|raison</exception>
+                        <exception regexp="yes">collection|semaine|manière|année|raison|série|ensemble|gamme|assortiment|lot|palette|variété|choix|répertoire|éventail</exception>
```
Comment (LucieSteib) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:96693]: suggestion(unblocking): panel, sélection, panoplie, catalogue, compilation, ensemble?

**Pull request merged**